package.json

```
@@ -9,7 +9,7 @@
     "test:watch": "jest --watchAll",
     "build": "ts-node --transpileOnly --logError ./esbuild.config.ts",
     "start": "nodemon",
-    "start:docker": "docker compose up --build"
+    "start:docker": "npm run build && docker compose up"
```

This should either use `concurrently` or `npm-run-all`, or just use a hook:
```suggestion
    "prestart:docker": "npm run build",
    "start:docker": "docker compose up"
```